### PR TITLE
Add integration tests to ci

### PR DESCRIPTION
**Pull request by DiFerMa**

### CODE REVIEW

**Review by phiwuu — COMMENTED** [review]
I believe the `MAKEFILE` needs an update since the introducion of `lobster-cpptest`.
Please have a look at https://github.com/bmw-software-engineering/lobster/blob/dda6fda8464fda00e9b2ae4fede9cde76674bd9c/Makefile#L25-L33 and add the following line:
```
make -C packages/lobster-cpptest
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,3 +71,20 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip --break-system-packages
+          python3 -m pip install -r requirements_dev.txt --break-system-packages
```
Comment (christophkloeffel) [inline .github/workflows/ci.yml:87]:
```suggestion
          python3 -m pip install --upgrade pip
          python3 -m pip install -r requirements_dev.txt
```

Comment (christophkloeffel) [reply .github/workflows/ci.yml:87]: since we are installing python explicitly via GitHub action there is no need for this argument to be present

**Review by christophkloeffel — CHANGES_REQUESTED** [review]

**Comment by christophkloeffel** [comment]
CI fails because of #89

**Comment by christophkloeffel** [comment]
will be replaced by #96

**Pull request closed without merging**